tools/setup/emoji/emoji_names.py

```diff
@@ -1378,7 +1378,7 @@
     '1f197': {'canonical_name': 'squared_ok', 'aliases': []},
     # from unicode, and to parallel Symbols/135. Saving up for Symbols/171
     '1f199': {'canonical_name': 'squared_up', 'aliases': []},
-    '1f192': {'canonical_name': 'cool', 'aliases': []},
+    '1f192': {'canonical_name': 'squared_cool', 'aliases': []},
```

Comment:
squared_cool can have a cool alias maybe?

Reply:
@gs0510 , thanks for the suggestion but Rishi Gupta suggested this ☺